webrtc.html

```diff
@@ -11181,9 +11181,10 @@ <h2>
                 <dfn data-idl="">setCodecPreferences</dfn>
               </dt>
               <dd>
+		<div id="setcodecpreferences-receive">
```

Comment:
I think I found one more place we forgot to upate. Should we also delete this sentence?

https://w3c.github.io/webrtc-pc/#dfn-final-steps-to-create-an-answer says
> If transceiver.[direction](https://w3c.github.io/webrtc-pc/#dom-rtcrtptransceiver-direction) is "[sendonly](https://w3c.github.io/webrtc-pc/#dom-rtcrtptransceiverdirection-sendonly)" or "[sendrecv](https://w3c.github.io/webrtc-pc/#dom-rtcrtptransceiverdirection-sendrecv)", exclude any codecs not included in the [list of implemented send codecs](https://w3c.github.io/webrtc-pc/#dfn-list-of-implemented-send-codecs) for kind.

Since it should be OK for pc1 to receive X and pc2 to receive Y, we shouldn't filter the preference on send codecs because that would exclude X from pc1's offer if X was a recvonly codec.

Reply:
That sentence needs to stay. Until we are ready to more officially deal with answer codecs that are not in the offer which JSEP allows.